tests/frontier/opcodes/test_call_and_callcode_gas_calculation.py

```diff
@@ -72,7 +73,10 @@ def callee_code(callee_opcode: Op) -> Bytecode:
         GAS <- value doesn't matter
         CALL/CALLCODE.
     """
-    return callee_opcode(Op.GAS(), 0xFEFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF, 1, 0, 0, 0, 0)
+    # The address needs to be random,
+    # otherwise the calculations (empty_account_cost) are incorrect.
+    address = f"0x{random.getrandbits(160):040x}"
```

Comment:
We try to avoid using the `random` library in order to guarantee that the fixtures are unchanged from one fill to the next.

The best approach here is to add `pre: Alloc` to the parameters of the fixture and then get an address using `pre.empty_account()`. This account is guaranteed to be empty but it's also calculated deterministically, so it will be the same over different fillings of the same test.

Reply:
Thanks, done.